src/features/modals/DeleteTrackModal/hooks/useDeleteTrackModal.ts

```diff
@@ -1,29 +1,33 @@
+import { useDispatch, useSelector } from 'react-redux'
 import { deleteTrack } from '../../../../api/track'
-import { useCardIdentifierValueProviderContext } from '../../../../context/CardIdentifierProvider'
-import { useLoaderProviderContext } from '../../../../context/LoaderProvider'
-import { useModalFuncContext } from '../../../../context/ModalProvider'
 import { showSuccess } from '../../../../shared/helpers/tosts/showSuccess'
+import {
+  actionClearCardId,
+  actionCloseDeleteTrackModal,
+  selectorCardId,
+  useMainLoading,
+} from '@/redux'
 
 function useDeleteTrackModal() {
-  const { trackId, clearTrackId } = useCardIdentifierValueProviderContext()
-  const { setDeleteTrackModal } = useModalFuncContext()
-  const { startLoading } = useLoaderProviderContext()
+  const dispatch = useDispatch()
+  const trackId = useSelector(selectorCardId)
```

Comment:
```suggestion
  const trackId = useSelector(selectCardId)
```
`selector` це функція, тому назва повинна починатися з дієслова, зазвичай з `select`
https://redux.js.org/usage/deriving-data-selectors

Reply:
виправлено